Buildings/Controls/OBC/Facade/ShadingEnable.mo

```diff
@@ -0,0 +1,121 @@
+within Buildings.Controls.OBC.Facade;
+block ShadingEnable "Generic shading device enable/disable seqence"
+
+  parameter Boolean use_solIrr = false
+    "Set to true if enabling based on solar irradiance, otherwise enabling based on temeprature"
+    annotation(Dialog(group = "Conditional"));
+  parameter Modelica.SIunits.Temperature TThr = 298.15
+    "Temperature threshold (either zone or outdoor air)"
+    annotation(Evaluate=true, Dialog(group="Hysteresis", enable = not use_solIrr));
+  parameter Modelica.SIunits.Irradiance irrThr = 1000
+    "Solar irradiance threshold"
+    annotation(Evaluate=true, Dialog(group="Hysteresis", enable = use_solIrr));
+
+  CDL.Interfaces.BooleanInput uEnable "Shading device enable schedule"
+    annotation (Placement(transformation(extent={{-120,-60},{-80,-20}}),
+    iconTransformation(extent={{-140,-80},{-100,-40}})));
+  CDL.Interfaces.RealInput T if not use_solIrr
+    "Zone or oudoor air temperature"
+    annotation (Placement(transformation(extent={{-120,20},{-80,60}}),
+    iconTransformation(extent={{-140,40},{-100,80}})));
+  CDL.Interfaces.RealInput irr if use_solIrr
+    "Total solar irradiance on horizontal or on window surface"
+    annotation (Placement(transformation(extent={{-120,-20},{-80,20}}),
+    iconTransformation(extent={{-140,-20},{-100,20}})));
+
+  CDL.Interfaces.BooleanOutput yShaEna
```

Comment:
Because the room model, and also the Spawn interface, uses a continuous signal from 0 to 1, please change this to a Real output. Then it is compatible with Buildings.ThermalZones.Detailed.Examples.MixedAirFreeResponse